02/app.js

```diff
@@ -1,23 +1,25 @@
 const randomArray = createRandomArray();
 console.log(randomArray);
 
-
+for (let i = 0; i < randomArray.length; i++) console.log(randomArray[i]);
```

Comment:
Jestem zwolennikiem używania nawiasów klamrowych każdym przypadku - jest to czytelniejsze i nic nie pominiemy przypadkowo :)